crates/api_common/src/lib.rs

```diff
@@ -348,6 +350,20 @@ pub async fn check_community_ban(
   }
 }
 
+pub async fn check_person_block(
+  person_id: PersonId,
+  recipient_id: PersonId,
+  pool: &DbPool,
+) -> Result<(), LemmyError> {
+  // Flip the recipient and the person
```

Comment:
Why flip it? Maybe change the param order in one place or the other?

Reply:
It either must be done here, or on the `::read`. I think I prefer to have it here, because then the `::read` uses the actual columns. A *check*, is kinda the reverse, you're checking to see if this person blocked you. But I'll change the params  to make it clearer.

Reply:
Ah I see. Maybe just include this explanation in the comment.